src/Commands/BuildToolsCommand.php

```diff
@@ -173,6 +594,163 @@ public function createBuildEnv(
         }
     }
 
+    /**
+     * Install the apporpriate CMS on the newly-created Pantheon site.
+     *
+     * @command build-env:install-site
+     */
+    public function installSite(
+        $site_env_id,
+        $siteDir = '',
+        $site_install_options = [
+            'account-mail' => '',
+            'account-name' => '',
+            'account-pass' => '',
+            'site-mail' => '',
+            'site-name' => ''
+        ])
+    {
+        // TODO: Detect WordPress sites and use wp-cli to install.
+        list($site, $env) = $this->getSiteEnv($site_env_id);
+
+        $this->log()->notice('Install site on {site}', ['site' => $site_env_id]);
+
+        // Set the target environment to sftp mode prior to installation
+        $this->connectionSet($env, 'sftp');
+
+        $command_line = "drush site-install --yes";
```

Comment:
I made this branch a month ago but forgot to open the PR apparently: https://github.com/pantheon-systems/example-drops-8-composer/pull/60

Different repos (WordPress, Acquia Lightning, etc) may have totally different install steps. I think we should let the given repo provide its own CMS installation instructions.